classes/controller/AdminController.php

```diff
@@ -476,6 +476,7 @@ public function __construct()
             $this->_defaultOrderBy = $this->identifier;
         }
         $this->tabAccess = Profile::getProfileAccess($this->context->employee->id_profile, $this->id);
+        $this->context->employee->setTabAccess($this->tabAccess);
```

Comment:
Seems weird. What if I navigate to another tab? Then employee automatically allowed to do anything? :)